lib/practice_exercises.rb

```diff
@@ -1,13 +1,31 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  index = 0
+  last_value = nil
+  list.each do
+    if list[index] == last_value
+      list.delete_at(index)
+    else
+      last_value = list[index]
+      index += 1
+    end
+  end
+  return list
 end
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n * m), where n is the iteration over the unknown prefix variable, and m is the iteration over each object in the strings array
+# Space Complexity: O(1)
```

Comment:
Since you are creating a new string which has a max size m, then I would say the space complexity is O(m)